tensorflow/python/ops/image_ops_impl.py

```diff
@@ -939,6 +939,13 @@ def resize_image_with_crop_or_pad(image, target_height, target_width):
   If `width` or `height` is smaller than the specified `target_width` or
   `target_height` respectively, this op centrally pads with 0 along that
   dimension.
+  
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
+    >> x = tf.random.normal(shape=(256, 256, 3))
```

Comment:
If you make a fake 4x4 image and resize it to be a 2x2 image I think you can use nice round numbers and have a testable example